doc-content/shared-kie-docs/src/main/asciidoc/KieServer/securing-passwords-proc.adoc

```diff
@@ -0,0 +1,95 @@
+[id='securing-passwords-proc_{context}']
+
+= Securing {KIE_SERVER} passwords with a keystore
+
+{KIE_SERVER} uses passwords to authenticate users. For increased security, you can store {KIE_SERVER} passwords in a keystore. Because it supports symmetrics keys, you must use a Java Cryptography Extension KeyStore (JCEKS) with {KIE_SERVER}. Use KeyTool, which is part of the JDK installation, to create a new JCEKS.
```

Comment:
"Because {KIE_SERVER} supports ..."  Otherwise it's a dangling modifier (would need to be "Because it supports ..., {KIE_SERVER} bla bla ...")